src/components/generics/vacation-test-forms/employeeVacationRequests.tsx

```diff
@@ -130,7 +131,7 @@ const ExpandableRow = (
                 <TableBody>
                   {Object.values(vacationRequests).map(request => (
                     <TableRow key={ request.id }>
-                      <TableCell component="th" scope="row">{ request.comment }</TableCell>
+                      <TableCell component="th" scope="row">{ request.message }</TableCell>
```

Comment:
Use of the scope attribute is triggering a problem in VSCode, does it need to be used? If so it could be worth using a comment to disable the linter in this case